packages/@webex/internal-plugin-metrics/src/rtcMetrics/index.ts

```diff
@@ -40,18 +58,23 @@ export default class RtcMetrics {
    * Initialize the interval.
    *
    * @param {object} webex - The main `webex` object.
-   * @param {string} meetingId - The meeting id.
+   * @param {IdType} Ids - Meeting or Calling id.
    * @param {string} correlationId - The correlation id.
    */
-  constructor(webex, meetingId, correlationId) {
+  constructor(webex, {meetingId, callId}: IdType, correlationId) {
```

Comment:
Why do we need to add a new type for this? We can just have 2 optional parameters here: meetingId, callId and based on what is passed while creating object of this class, rtctstats will be sent with that

Reply:
That can also be done, I have kept them in one object. Both are optional.